proposed/project-file-simplification.md

```diff
@@ -0,0 +1,351 @@
+# Project file simplification and improved change management
+
+// Contents of Type should match template name as much as possible
+
+## Summary
+[summary]: #summary
+
+This proposal is to simplify the project file while giving the user better control over breaking changes and new warnings.
+
+The approach is versioned default properties specified by a `Base` attribute in the project file, a `Directory.Build.props`, or an `Import` file. This attribute specifies a version of the file containing the default properties. The project file is further simplified by `Type` attribute on the `Project` element. The `Type` is not versioned, although versioned default properties can be added as part of the `Base` evaluation. The `Type` attribute replaces the current `SDK` and the `Type` file contains both the `SDK` name and properties that apply to the application `Type`.
```

Comment:
A worry I have about `Type` is that it's likely going to create _dialects_ of SDK projects. Essentially, are you using old school `Sdk` or new `Type`. I'm not sure the value of this attribute outweighs the cost of having competing dialects in the ecosystem. 

I don't have the same worry about `Base`. I see that as say `SdkVersion` which is something that large repos will care about and will be in centralized places, much like say CPM is today. It will be a _big repo_ property but likely won't impact most .NET development. An _expert_ feature.